aws/ec2/test_ec2_ebs_snapshots_are_private.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+
+from aws.ec2.resources import ec2_ebs_snapshots_create_permission
+from aws.ec2.helpers import is_ebs_snapshot_public
+
+
+@pytest.mark.ec2
+@pytest.mark.parametrize(
+    "ec2_ebs_snapshot",
+    ec2_ebs_snapshots_create_permission(),
+    ids=lambda ebs: ebs["SnapshotId"],
+)
+def test_ec2_ebs_snapshot_are_private(ec2_ebs_snapshot):
+    assert not is_ebs_snapshot_public(
+        ec2_ebs_snapshot
+    ), "Snapshot {} is publicly accessible.".format(ec2_ebs_snapshot["SnapshotId"])
```

Comment:
Do we need the ownerid for processing the report?

Reply:
No, snapshot id should be enough. This is also consistent with most of our other AWS/GCP test output.

Reply:
Just curious - how do you know where to file the issue? Wouldn't you have to look that up?

Reply:
I filed #286 as a follow-up to this, but essentially for our use of pytest-services, we are only ever running the test suite against a specific account. This is true locally and in our nightly scans. If you look at the Bugzilla bugs (can slack you one), it specifies the aws account name and region in the finding.

Reply:
Got it - I forgot about the args being in the formatted output.